app/views/home/index.html.erb

```diff
@@ -1,3 +1,36 @@
-<h1> home index </h1>
-
-<%= link_to "Login", login_path %>
+<div class="container-fluid">
+  <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <!-- <img class= src="/images/g" alt="First slide"> -->
+        <%= image_tag("basket_2.jpg", class: "d-block w-80 img-fluid") %>
+        <div class="carousel-caption d-none d-md-block leitbild-text">
+          <h3>Baskets</h3>
+        <p>Here are all the baskets!</p>
+        </div>
+      </div>
+      <div class="carousel-item">
```

Comment:
if anyone wants to change the pictures go for it, just make sure it put it in the assets/images folder.